postcss.config.js

```diff
@@ -1,3 +1,39 @@
+const platformPostcssConfig = require('@hashicorp/platform-postcss-config')
+
+/**
+ * TODO: this is a spike to show that this configuration may resolve
+ * the issues we're having in dev-dot when using @custom-media.
+ * We likely want to incorporate this into `@hashicorp/platform-postcss-config`.
+ * Eg, maybe it should accept an optional "options" object, where we can
+ * pass additional files to `importFrom`?
+ */
+function alsoImportDevDotCustomMedia(postcssConfig) {
+  const newPlugins = postcssConfig.plugins.map((plugin) => {
+    // we only want to modify postcss-preset-env
+    const isPresetEnv =
+      Array.isArray(plugin) &&
+      plugin.length == 2 &&
+      plugin[0] == 'postcss-preset-env'
+    if (!isPresetEnv) return plugin
```

Comment:
Does ESLint not run on this file? I would expect an error to happen here for the `curly` rule. CC @BRKalow

Reply:
Weird, yeah, it looks like this file isn't getting picked up.